ffcx/codegeneration/definitions.py

```diff
@@ -190,7 +192,7 @@ def _define_coordinate_dofs_lincomb(
         iq_symbol = self.symbols.quadrature_loop_index
         ic = create_dof_index(tabledata, ic_symbol)
         iq = create_quadrature_index(quadrature_rule, iq_symbol)
-        FE, tables = self.access.table_access(tabledata, self.entitytype, mt.restriction, iq, ic)
+        FE, tables = self.access.table_access(tabledata, self.entitytype, mt.restriction, iq, ic, 0)
```

Comment:
Maybe we should add a note here stating that due to the way we define the mixed dimensional code, i.e. that the integration domain for codim=1 has to be the "parent mesh", the coordinate element is  always of codim 0.